web/i18n/language.ts

```diff
@@ -74,8 +74,8 @@ export const languages = [
   {
     value: 'ko-KR',
     name: '한국어 (대한민국)',
-    example: '안녕, Dify!',
-    supported: false,
+    example: '안녕하세요, Dify!',
+    supported: true,
```

Comment:
I modified the boolean value to 'true' according to the i18n translation contribution guidelines.